packages/core/src/components/badge/stories/badge.args.ts

```diff
@@ -0,0 +1,36 @@
+export const BadgeStoryArgs = {
+  decorators: [],
+  parameters: {
+    actions: {
+      handles: [],
+    },
+    docs: {
+      description: {
+        component:
+          'Badges are inline block elements that usually appear near another element. Read the [Ionic documentation](https://ionicframework.com/docs/api/Badge) for more information about this component.',
+      },
+    },
+  },
+  argTypes: {
+    type: {
+      control: 'select',
+      options: [
+        'primary',
+        'secondary',
+        'tertiary',
+        'success',
+        'warning',
+        'danger',
+        'light',
+        'dark',
+      ],
+      defaultValue: { summary: 'primary' },
+      description:
+        'The type of the badge, the component will receive the color according to its type.',
+    },
+    children: {
+      control: 'text',
+      description: 'The content of the badge',
+    },
```

Comment:
**nit-pick**: others docs that has the same behavior call this a label, just to keep the same behavior on the whole doc, could you switch this?
Below example is on the button component, but you can see the same on chip
<img width="1125" alt="Captura de Tela 2023-08-17 às 17 55 15" src="https://github.com/juntossomosmais/atomium/assets/6757777/18a198e5-ed32-4a1a-9f2b-4874a8679033">

Reply:
Nice!! I agree with keeping it as a label! 
Children seems to me to be more related to react or vue code.

Made: https://github.com/juntossomosmais/atomium/pull/239/commits/7bcbb95f5ef9d64a1aa8f3623a8c6af9f2caaee8